mozilla-services/consvc-shepherd
# [SHEP-2] UI Created to Split a Single Campaign into Multiple Campaigns

## References

[SHEP-2](https://mozilla-hub.atlassian.net/browse/SHEP-2)

## Problem Statement

With the automatic creation of a child campaign when new booster deals are imported, we need a UI that allows to split a single campaign into multiple campaigns.

## Proposed Changes

Split Campaign Form Added to Allow Users to Split a Single Campaign into Multiple Campaigns.

## Verification Steps

1. Click on the "Split Campaign" icon on the Campaigns list page.
2. Click on the plus icon to split the campaign into multiple campaigns.
3. Add details for the newly created campaign.
4. Ensure the total net spend for the campaigns matches the actual value of the campaign before the split.

## Visuals

https://github.com/user-attachments/assets/0d9c82ce-3e5e-4c39-a891-96ad19c399e1



[SHEP-2]: https://mozilla-hub.atlassian.net/browse/SHEP-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
